zcbor/zcbor.py

```diff
@@ -3019,7 +3143,7 @@ def relativify(p):
 target_include_directories({target_name} PUBLIC
     {(linesep + "    ").join(((str(relativify(f)) for f in include_dirs)))}
     )
-"""
+{f"\n{smart_search}\n" if add_smart_search else ''}"""
```

Comment:
Nit: The use of backslashes in the formatted string doesn't work on version 3.10 python - forced me to update python to pickup this change.